### ISSUE TITLE
Fires multiple times when a dash "-" split key is used instead of a plus

### ISSUE THREAD

**Issue by meglio** [JohannesKlauss/react-hotkeys-hook#529]
This version works fine:

```js
useHotkeys(
        'command+shift+enter,ctrl+shift+enter,ctrl+alt+shift+left,command+option+shift+left,ctrl+alt+shift+right,command+option+shift+right',
        function (event, handler) {
            event.preventDefault();
            console.log(handler.key, event);

            // Returning false stops the event and prevents default browser events
            return false;
        }, {
            // Options
            enableOnTags: ['INPUT', 'TEXTAREA', 'SELECT'],
            keyup: false,
            keydown: true,
            splitKey: '+',
    }, [

    ]);
```

However, if you set `splitKey: '-'` and update the hotkeys string accordingly, it will fire the same event multiple times, which is reported by the `console.log()` in the handler.

Not sure if it's an issue with the hook or with the underlying hotkeys library though. Could you take a look at this please?

**Comment by JohannesKlauss** [comment]
@meglio Thx for the issue report, I'll have a loop as soon as possible.

**Comment by JohannesKlauss** [comment]
Fixed in version 3.3.2
